pkg/vm/vm.go

```diff
@@ -472,7 +500,9 @@ func (v *VM) Step() error {
 
 // step executes one instruction in the given context.
 func (v *VM) step(ctx *Context) error {
+	instruction_offset := v.Context().nextip
```

Comment:
In Go we use `camelCase`. Short-lived variables should have small names, how about `ip := ` or `offset`?

Reply:
`vm.go` shouldn't be reviewed here as the branch is based on #3640

Reply:
@Slava0135, please, see also the comments from https://github.com/nspcc-dev/neo-go/pull/3460#discussion_r1627461729. Let's firstly update and merge the #3460 and then move on with this PR.